src/plugins/intel_gpu/src/graph/kernel_impl_params.cpp

```diff
@@ -32,7 +34,7 @@ size_t kernel_impl_params::hash() const {
 }
 
 bool kernel_impl_params::operator==(const kernel_impl_params& rhs) const {
-    if (*desc != *rhs.desc)
+    if (desc != nullptr && rhs.desc != nullptr && *desc != *rhs.desc)
```

Comment:
what if one desc is nullptr and another is not?

Reply:
Fixed it.